config/config.go

```diff
@@ -128,6 +129,46 @@ func (cfg *AuctionTimeouts) validate(errs configErrors) configErrors {
 	return errs
 }
 
+func (data *ExternalCache) validate(errs configErrors) configErrors {
+	if data.Host == "" {
+		if data.Path != "" {
+			// Throw an error because we have a blank Host nad a non-blank Path
+			errs = append(errs, fmt.Errorf("Could not parse URL with empty Host: %s \n", data.Host+data.Path))
+		}
+		// Blank host and path are valid and will not throw an error
+		return errs
+	}
+
+	// Add "/" suffix to the path if missing
+	if data.Path != "" && !strings.HasPrefix(data.Path, "/") && !strings.HasSuffix(data.Host, "/") {
+		data.Path = "/" + data.Path
+	}
+
+	// Add "//" suffix hosts that don't come with a scheme
+	if strings.Index(data.Host, "//") == -1 {
```

Comment:
Hosts should not include a protocol/scheme or start with a "//". Not a bad idea to reverse the check and return a validation error.

Reply:
This is just in preparation to `url.Parse()`. After some trial and error I found out that function `url.Parse()` takes a URL that looks like `//www.example.com/somepath/v1/` and will return:
```
u.Scheme | ""
u.Host   | "www.example.com"
u.Path   | "/somepath/v1"
err      | nil
```
But, and this is the sole reason to add that extra `//`, if the string we pass to `url.Parse()` looks like `www.example.com/somepath/v1/`, it will surprisingly (or maybe not so much) return:
```
u.Scheme | ""
u.Host   | ""
u.Path   | "www.example.com/somepath/v1"
err      | nil
```
Apparently `url.Parse()` needs those two characters (`//`) to delimit the host.